### DISCUSSION TITLE
Don't display time in blog post date

### DISCUSSION THREAD

**Discussion by SeanTAllen** [squidfunk/mkdocs-material#6996] [Q&A] [answered]
If you use a full date format like:

`date: 2024-03-31T07:00:06-04:00`

So that you can do ordering based on time if there are multiple posts on the same day. There doesn't appear to be a way to not display the time as part of the blog post

`March 31, 2024, 7:00:06 AM -0400`

for the above with `post_date_format: long`

Is there a way to not display the time while still preserving the data for ordering?

**Comment by alexvoss** [comment]
The [documentation](https://squidfunk.github.io/mkdocs-material/plugins/blog/?h=post+date+format#config.post_date_format) suggests that you can create your own format using the [Babel pattern ](https://babel.pocoo.org/en/latest/dates.html#pattern-syntax).

**Reply by squidfunk** [reply]
Correct, ordering is done using the raw, unformatted `datetime` object as parsed from YAML:

https://github.com/squidfunk/mkdocs-material/blob/e71f00b4c9bc483d5ce60e91e5fd461305e389eb/src/plugins/blog/plugin.py#L135

You can then change all format strings to your liking.

**Comment by SeanTAllen** [comment]
Thanks

**Comment by squidfunk** [comment, marked as answer]
This was a regression reported in https://github.com/squidfunk/mkdocs-material/issues/7001, introduced in [9.5.16](https://github.com/squidfunk/mkdocs-material/releases/tag/9.5.16) and fixed in [9.5.17](https://github.com/squidfunk/mkdocs-material/releases/tag/9.5.17). The time will now, by default, not be included again.
